main.py

```diff
@@ -1,10 +1,45 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_data in data.items():
+        race_data = player_data.get("race")
+        guild_data = player_data.get("guild")
+        skills_data = race_data.get("skills")
+
+        race, created = Race.objects.get_or_create(
+            name=race_data.get("name"),
+            description=race_data.get("description")
+        )
+
+        guild = None
+        if guild_data:
+            guild, created = Guild.objects.get_or_create(
```

Comment:
Since you do not utilize the `created` variable, you could omit it using `_` here and on lines 30 and 17 to not allocate a variable to it.